### PR TITLE
fix: correct Brazilian portuguese translations

### PR DESCRIPTION
**Pull request by treasuryesc**
> Please provide enough information so that others can review your pull request:
I've sync'ed Brazilian Portuguese (pt-BR.csv) translations based on parent pt.csv file, adapting it for local terms and expressions.

> Explain the **details** for making this change. What existing problem does the pull request solve?
Several words and phrases were missing in previous file and some Modules didn't have translation for Brazilian Portuguese (pt-BR), so ERPNext was getting translations from parent language file (pt.csv)

> Screenshots/GIFs

**Todo:**

- [x]  delete **pt_br.csv**

### CODE REVIEW

**Comment by treasuryesc** [comment]
Hey guys, could you please take a look at this?
@phot0n help ! :)

**Comment by treasuryesc** [comment]
Hi @marination , I saw that you have merged some updates in translations lately. Could you please take a look at my PR? 
Thank you!

**Comment by treasuryesc** [comment]
> Hi @marination , I saw that you have merged some updates in translations lately. Could you please take a look at my PR? Thank you!

I've changed my commit message, I forgot to put "fix" on it

**Comment by marination** [comment]
@treasuryesc this PR will need a couple of changes. Will deal with them one at a time.
Firstly from what I see for language code **pt-BR**, the following files are used to get translations (i added print statements in `load_lang`):
Screenshot 2022-07-20 at 12 23 00 PM

Seems like **pt_br** is a legacy file which can be deleted. Once that is done this can be merged

**Todo:**

- [ ]  delete **pt_br.csv**

**Comment by treasuryesc** [comment]
> * [x]   delete **pt_br.csv**

@marination , done

File: erpnext/translations/pt-BR.csv [hunk 1]
```diff
@@ -1,3050 +1,9840 @@
...
-apps/erpnext/erpnext/setup/doctype/company/company.js +126,Successfully deleted all transactions related to this company!,Todas as transações relacionadas a esta empresa foram excluídas com sucesso!
-apps/erpnext/erpnext/accounts/report/accounts_payable/accounts_payable.js +22,As on Date,Como na Data
-apps/erpnext/erpnext/setup/setup_wizard/operations/install_fixtures.py +78,Probation,Provação
-apps/erpnext/erpnext/accounts/doctype/sales_invoice/sales_invoice.js +866,Return / Credit Note,Devolução / Nota de Crédito
-apps/erpnext/erpnext/public/js/controllers/transaction.js +322, does not exist!,não existe!
-apps/erpnext/erpnext/accounts/report/accounts_receivable/accounts_receivable.html +137,Total Paid Amount,Valor total pago
-DocType: Job Card,Transferred Qty,Qtde Transferida
-apps/erpnext/erpnext/setup/setup_wizard/operations/install_fixtures.py +141,Planning,Planejamento
-apps/erpnext/erpnext/accounts/report/purchase_register/purchase_register.py +80,Supplier Id,ID do Fornecedor
-apps/erpnext/erpnext/manufacturing/doctype/bom/bom.py +290,Quantity should be greater than 0,Quantidade
```
Comment (marination) [inline erpnext/translations/pt-BR.csv]:
```suggestion
'Update Stock' cannot be checked for fixed asset sale,"'Atualizar Estoque' não pode ser selecionado para venda de ativo fixo",
```

**Review by marination — APPROVED** [review]

**Pull request merged**